lms/djangoapps/grades/tasks.py

```diff
@@ -24,9 +28,27 @@
 
 @task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
 def recalculate_subsection_grade(
+        # pylint: disable=unused-argument
         user_id, course_id, usage_id, only_if_higher, weighted_earned, weighted_possible, **kwargs
 ):
     """
+    Shim to allow us to modify this task's signature without blowing up production on deployment.
+    """
+    recalculate_subsection_grade_v2.apply(
+        kwargs=dict(
+            user_id=user_id,
+            course_id=course_id,
+            usage_id=usage_id,
+            only_if_higher=only_if_higher,
+            expected_modified_time=kwargs.get('expected_modified_time', 0),  # Use the unix epoch as a default
+            score_deleted=kwargs['score_deleted'],
+        )
+    )
+
+
+@task(default_retry_delay=30, routing_key=settings.RECALCULATE_GRADES_ROUTING_KEY)
+def recalculate_subsection_grade_v2(**kwargs):
```

Comment:
@jibsheet This is the change we've discussed to prevent blowing up worker error reporting while deploying. I'd almost rather avoid the `_vN` in the signature and just do a one-time shim to get us to a clean `recalculate_subsection_grade(**kwargs)` which can be kept backwards compatible internally, how do you feel?